### PR TITLE
Walkthrough part five

### PR DESCRIPTION
**Pull request by swalsh15**
Updated doPost to create entity from comment and store into datastore. No querying yet so fetching comments doesn't won't work until the next part. This is part five of the walkthrough.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -23,21 +26,33 @@
...
     response.setContentType("application/json;");
     response.getWriter().println(json);
   }
 
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String comment = request.getParameter("text_input");
+    String name = request.getParameter("name_input");
+    if (comment == null || name == null) {
+      response.sendRedirect("/index.html");
```
Comment (g-harel) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:47]: Nit: An interesting stretch goal could be to provide some feedback to the user (whether it failed or not).

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -39,17 +39,13 @@ function highlightTab() {
...
-            document.getElementById("data-container").innerText += list[i] + "\n";
-        }
-    }));
+function fetchComments() {
+  (fetch('/comments').then(response => response.json()).then((comments) => {
+    for (const comment of comments) {
+      let div = document.createElement("div");
+      div.textContent = comment;
+      div.setAttribute("class", "comment");
+      document.getElementById("data-container").appendChild(div);
```
Comment (g-harel) [inline portfolio/src/main/webapp/script.js:48]: 👏

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -39,17 +39,13 @@ function highlightTab() {
...
-    (fetch('/data').then(response => response.json()).then((list) => {
-        console.log(list);
-        for (let i = 0; i < list.length; i++) {
-            document.getElementById("data-container").innerText += list[i] + "\n";
-        }
-    }));
+function fetchComments() {
+  (fetch('/comments').then(response => response.json()).then((comments) => {
+    for (const comment of comments) {
+      let div = document.createElement("div");
```
Comment (augerm) [inline portfolio/src/main/webapp/script.js:45]: let --> const

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -86,4 +86,9 @@ main {
 
 .divider {
   border-top: 3px dashed #bbb;
-}
\ No newline at end of file
+}
+
+.comment {
+  box-shadow: 10px 10px 5px #aaaaaa;
```
Comment (augerm) [inline portfolio/src/main/webapp/style.css:92]: Appreciate the box-shadow usage :)

**Pull request merged**